docs/reference/tctl/cluster/list-search-attributes.md

```diff
@@ -0,0 +1,15 @@
+---
+id: list-search-attributes
+title: tctl cluster list-search-attributes
+sidebar_label: list-search-attributes
+description: How to list all Search Attributes using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+The `tctl cluster list-search-attributes` command lists all [Search Attributes](/docs/content/what-is-a-search-attribute) that can be used in the `--query` modifier of the [`tctl workflow list`](/docs/reference/tctl/workflow/list) command.
```

Comment:
```suggestion
The `tctl cluster list-search-attributes` command lists all [Search Attributes](/docs/content/what-is-a-search-attribute) that can be used in the 
`--query` modifier of the [`tctl workflow list`](/docs/reference/tctl/workflow/list) command.
```

Reply:
because in the UI, the '--query' breaks across two lines such that `-` ends line one and `-query` starts the next line. may not be a big deal, but at first glance it looks strange.

Reply:
Interesting! Which browser are you using, @aarohib?

Reply:
Behavior confirmed in Safari and Brave on macOS and Edge on Windows. No repro in Firefox on either macOS or Windows. (Also no repro in Internet Explorer, but who cares?)

When repro'd, line break occurs before `--query`, between the hyphens, immediately following the hyphens, or after `--query`, depending on window width.

Because this is a rendering issue and not a source-file issue, I'm gonna stet rather than hard-coding a line break.

This appears to be consistent behavior (based on my extensive sampling during the past five minutes). Let's check with Candace or Shawn to find out whether they know a way to override this behavior in CSS without preventing desired word breaks.